### ISSUE TITLE
VnV Plan Review: Tests for Functional Requirements section

### ISSUE THREAD

**Issue by samm82** [balacij/Drasil#34]
# TODO
*Any point that requires more discussion should be pulled out to a different issue; I grouped these all here since they are related and small.*

- [x] There shouldn't be a comma after "categories". 
- [x] The description of the Testing Inputs Are Outputted Accurately section is quite confusing.
   - [ ] If kept the way it is, it should have a comma after "testing".
- [x] Do you mean to exclude $0$ and $L_B$ as values of $x$ when testing the BVP solver?
- [x] It should be "The expected outputs **will be** confirmed...."
- [x] What ranges of values of $E_B$ and $I_B$ will be chosen?
   - [x] I would rephrase to avoid saying " $E_B$ s".

**Comment by balacij** [comment]
3. If I did, I don't remember why and didn't explain it there. Good catch.

Thanks! 758ec77d2
